test/methods.js

```diff
@@ -124,7 +124,32 @@ describe('nemo-view @methods@', function () {
       done();
     }, util.doneError(done));
   });
-
+  it('should return true using @ValidateText@Positive@ method', function (done) {
+    nemo.view.simple.pageHeaderValidateText('Sample form stuff').then(function (actual) {
+      assert.equal(actual, true);
+      done();
+    }, util.doneError(done));
+  });
+  it('should return false using @ValidateText@negative@ method', function (done) {
+    nemo.view.simple.pageHeaderValidateText('form stuff').then(function () {
+      done(new Error('couldn\'t find text: form stuff'));
```

Comment:
The error here would be that the method resolved it's promise when it shouldn't have. I.e. the message should be "resolved promise as true when we should have rejected it" or something like that